applications/ParticleMechanicsApplication/python_scripts/deprecated_python_scripts/deprecated_particle_mpm_solver.py

```diff
@@ -0,0 +1,370 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
this only shows up as change bcs you renamed it right?
=> hence I wont comment on it

Reply:
Yup! make it deprecated